pkg/build/util/util_test.go

```diff
@@ -163,8 +164,7 @@ func TestSafeForLoggingS2IConfig(t *testing.T) {
 	if redactedRegex.MatchString(config.ScriptDownloadProxyConfig.HTTPSProxy.String()) {
 		t.Errorf("credentials stripped from original scripts proxy: %v", config.ScriptDownloadProxyConfig.HTTPSProxy)
 	}
-	//checkEnvList(t, config.Environment, true)
-
+	checkEnvList(t, config.Environment, true)
```

Comment:
we should also make sure none of the other proxy fields changed in the original, e.g. ScriptDownloadProxyConfig.HTTPProxy, etc.